.github/workflows/e2e-soak-trigger.yaml

```diff
@@ -0,0 +1,49 @@
+name: E2ESoakTrigger
+on:
+  schedule:
+    - cron: '0 */3 * * *'
+jobs:
+  resolve_cluster:
+    if: github.repository == 'aws/karpenter'
+    runs-on: ubuntu-latest
+    outputs:
+      PREEXISTING_CLUSTERS: ${{ steps.list_clusters.outputs.PREEXISTING_CLUSTERS }}
+    steps:
+    - uses: actions/checkout@v4
+    - name: configure aws credentials
+      uses: aws-actions/configure-aws-credentials@010d0da01d0b5a38af31e9c3470dbfdabdecca3a
+      with:
+       role-to-assume: arn:aws:iam::${{ vars.ACCOUNT_ID }}:role/${{ vars.ROLE_NAME }}
+       aws-region: eu-north-1
+    - uses: actions/setup-go@v4
+      with:
+        go-version-file: test/hack/soak/go.mod
+        cache-dependency-path: test/hack/soak/go.sum
+        check-latest: true
+        cache: false
+    # Grab all the names of the soak testing clusters, and identify clusters that will need to be deleted
+    # Empty cluster_name will indicate that a new cluster will be created
+    - id: list_clusters 
+      name: "Run list clusters script"
+      run: |
+        PREEXISTING_CLUSTERS="$(go run main.go)" 
+        echo PREEXISTING_CLUSTERS="$PREEXISTING_CLUSTERS" >> "$GITHUB_OUTPUT"
+      working-directory: ./test/hack/soak
+      shell: bash
+  soak:
+    permissions:
+      id-token: write # aws-actions/configure-aws-credentials@v4.0.1
+    needs: [resolve_cluster]
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.resolve_cluster.outputs.PREEXISTING_CLUSTERS) }}
+    uses: ./.github/workflows/e2e.yaml
+    with:
+      suite: Integration
```

Comment:
Is there value in factoring this out to be one or multiple suites? I can imagine us wanting to do more here, especially since the name Integration is a little misleading since it doesn't test integration of all things.